modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -0,0 +1,124 @@
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	loginTest(),
+	localizationPageTests,
+	objectPagesTest
+);
+
+let createObjectDefinitionId;
+
+test.afterEach(async ({apiHelpers, localizationInstanceSettingsPage, page}) => {
+	await apiHelpers.objectAdmin.deleteObjectDefinition(
+		createObjectDefinitionId
+	);
+
+	await page.goto('/');
+
+	await page.getByLabel('Abrir menu de aplicativosCtrl+Alt+A').click();
+
+	await page.getByRole('tab', {name: 'Painel de Controle'}).click();
+
+	await page
+		.getByRole('menuitem', {name: 'Configurações da Instância'})
+		.click();
+
+	await page.getByRole('link', {name: 'Localização'}).click();
+
+	await localizationInstanceSettingsPage.moveLanguageToCurrent(
+		'Disponível',
+		'en_US'
+	);
+
+	await page.getByLabel('Idioma padrão').selectOption('en_US');
+
+	await page.getByRole('button', {name: 'Salvar'}).click();
+});
+
+test.describe('Manage object fields through Model Builder', () => {
+	test('can add picklist object field to object definition node', async ({
+		apiHelpers,
+		localizationInstanceSettingsPage,
+		page,
+	}) => {
+		await localizationInstanceSettingsPage.goto();
+
+		await localizationInstanceSettingsPage.setDefaultLanguage(
+			'Default Language',
+			'pt_BR'
+		);
+
+		await localizationInstanceSettingsPage.moveLanguageToAvailable(
+			'Current',
+			'en_US'
+		);
+
+		const objectDefinitionExternalReferenceCode =
+			'ObjectDefinition' + getRandomInt();
+
+		const requestBody = {
+			active: true,
+			externalReferenceCode: objectDefinitionExternalReferenceCode,
+			label: {
+				pt_BR: objectDefinitionExternalReferenceCode,
+			},
+			name: objectDefinitionExternalReferenceCode,
+			objectFields: [
+				{
+					DBType: 'String',
+					businessType: 'Text',
+					externalReferenceCode: 'textField',
+					indexed: true,
+					indexedAsKeyword: false,
+					indexedLanguageId: '',
+					label: {pt_BR: 'textField'},
+					listTypeDefinitionId: 0,
+					name: 'textField',
+					required: false,
+					system: false,
+					type: 'String',
+				},
+			],
+			objectFolderExternalReferenceCode: 'default',
+			panelCategoryKey: 'control_panel.object',
+			pluralLabel: {
+				pt_BR: objectDefinitionExternalReferenceCode,
+			},
+			scope: 'company',
+			status: {code: 0},
+		};
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postObjectDefinition(requestBody);
+
+		createObjectDefinitionId = objectDefinition.id;
+
+		await page.getByLabel('Abrir menu de aplicativosCtrl+Alt+A').click();
+		await page
+			.getByRole('menuitem', {
+				name: objectDefinitionExternalReferenceCode,
+			})
+			.click();
+		await page
+			.getByText('Adicionar ' + objectDefinitionExternalReferenceCode)
+			.click();
+		await page.getByLabel('textField').fill('test');
+		await page.getByRole('button', {name: 'Salvar'}).click();
+
+		expect(
+			page.getByText('Solicitação concluída com sucesso.')
+		).toBeVisible();
```

Comment:
Relying purely on success messages is not so good, because sometimes they display but the actual thing did not happen. Keep this assertion here, but add steps to go back to the entries list and assert that the entry is really there.